File: 7-kyu/spraying-trees/python/solution.py
def task(w, n, c):
    worker_lookup = {'Monday': 'James', 'Tuesday': 'John', 'Wednesday': 'Robert', 'Thursday': 'Michael', 'Friday': 'William'}
    return f'It is {w} today, {worker_lookup[w]}, you have to work, you must spray {n} trees and you need {n * c} dollars to buy liquid'

File: 7-kyu/spraying-trees/javascript/solution.js
const task = (w, n, c) => {
  const workerLookup = {"Monday": "James", "Tuesday": "John", "Wednesday": "Robert",
                        "Thursday": "Michael", "Friday": "William"};
  return `It is ${w} today, ${workerLookup[w]}, you have to work, you must spray ${n} trees and you need ${n * c} dollars to buy liquid`;
}